modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/dto/v1_0/converter/SkuDTOConverter.java

```diff
@@ -117,6 +111,21 @@ public Sku toDTO(DTOConverterContext dtoConverterContext) throws Exception {
 		return options;
 	}
 
+	private void _setSkuReplacement(
+			Sku sku, long replacementCProductId,
+			String replacementCPInstanceUuid)
+		throws Exception {
+
+		CPInstance replacementCPInstance =
+			_cpInstanceService.fetchCProductInstance(
+				replacementCProductId, replacementCPInstanceUuid);
+
+		sku.setDiscontinuedDate(replacementCPInstance.getDiscontinuedDate());
```

Comment:
We can do these setters with lambda functions instead and return the object (Sku) directly without creating a variable